app/views/merchant_invoices/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1 id="merchant_name"><%= "#{@merchant.name} Invoices" %></h1>
+
+<h2> Invoices: </h2>
+<ul id="merchant_invoices">
+  <% @merchant.invoices.distinct.each do |invoice|%>
+    <li> Invoice: <%=link_to "#{invoice.id}", "/merchants/#{@merchant.id}/invoices/#{invoice.id}"%>
```

Comment:
A good idea to refactor here would be to use advanced routes, `merchant_invoice_path(merchant_id, invoice_id)`. But this looks great!!